index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Car Lot</title>
+
+    <!-- Bootstrap -->
+    <link href="/lib/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
The leading `/` is not necessary, and in cases like mine it will not run if it's there.